volontulo/templates/common/header.html

```diff
@@ -14,7 +14,7 @@
         <div class="col-xs-12">
             <h1 class="navbar-header">
                 <a href="{% url 'index' %}">
-                    <img src="{% static "volontulo/img/volo_logo.svg" %}" alt="Volontulo">
+                    <img src="{% static "volontulo/img/volo_logo.svg" %}" alt="Volontulo" />
                     Volontulo
```

Comment:
if we have logo already with name of project, we don't need to duplicate it using text - this occurance of Volontulo is ready to be deleted.